swri_transform_util/CMakeLists.txt

```diff
@@ -60,6 +60,11 @@ include_directories(SYSTEM
   ${PROJ_INCLUDE_DIRS}
 )
 
+if ($ENV{ROS_DISTRO} STREQUAL "kinetic")
+  set(CMAKE_CXX_STANDARD 11)
```

Comment:
This should be set per target, not globally

Reply:
CMAKE_CXX_STANDARD is global per project, so setting the standard for every target in the package does effectively the same thing I believe. I was just trying to save a few keystrokes.

I'm happy to make the change just to be consistent stylistically across all of our other packages.